Elsewhen/Elements/TimeZoneChoice/TimezoneChoiceView.swift

```diff
@@ -152,7 +157,7 @@ struct TimezoneChoiceView: View {
     }
     
     var body: some View {
-        if DeviceType.isPadAndNotCompact {
+        if isPadAndNotCompact {
```

Comment:
What was wrong with using the helper on DeviceType?

Reply:
It didn't adapt to transitions, so if you moved the app to SlideOver, it would still return "not compact"

Reply:
Yes, that is a better approach. I'm not very confident with protocols in Swift, so I started it but shied away from it. I'll give it a go :)